src/script/script.cpp

```diff
@@ -238,11 +240,11 @@ bool CScript::IsPushOnly() const
     return true;
 }
 
-std::string CScript::ToString() const
```

Comment:
I agree with you, I'm going to put the prefixes back to reduce the diff in the next commit. Thanks for the info and for reviewing these changes.